### ISSUE TITLE
Taking to Survey causing crash (2.8.1)

### ISSUE THREAD

**Issue by jamztang** [mixpanel/mixpanel-iphone#279]
I'm setting up Mixpanel as a dependancy of ARAnalytics using Cocoapods (0.37.2), is it something on the radar?

Other specs:
Xcode 6.3.1
Simulator iPhone 6 iOS 8.3 (12F69)

```
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: 'Could not find a storyboard named 'MPSurvey' in bundle NSBundle </Users/james/Library/Developer/CoreSimulator/Devices/BA4646B9-6CE0-4BB0-8474-847311AE12B3/data/Containers/Bundle/Application/086F3357-147A-409F-905F-399EEAFF51DE/Today.app> (loaded)'
*** First throw call stack:
(
	0   CoreFoundation                      0x0000000105057c65 __exceptionPreprocess + 165
	1   libobjc.A.dylib                     0x00000001041f9bb7 objc_exception_throw + 45
	2   UIKit                               0x00000001060866b8 -[UIStoryboard name] + 0
	3   Mixpanel                            0x00000001037603f1 -[Mixpanel presentSurveyWithRootViewController:] + 161
	4   Mixpanel                            0x0000000103760dec __43-[Mixpanel showSurveyWithObject:withAlert:]_block_invoke915 + 172
	5   UIKit                               0x0000000105db49e1 -[UIAlertController _fireOffActionOnTargetIfValidForAction:] + 55
	6   UIKit                               0x0000000105db4f23 __85-[UIAlertController _dismissAnimated:triggeringAction:triggeredByPopoverDimmingView:]_block_invoke + 30
	7   UIKit                               0x0000000105c588f9 -[UIPresentationController transitionDidFinish:] + 1118
	8   UIKit                               0x0000000105c5ab7c __56-[UIPresentationController runTransitionForCurrentState]_block_invoke_2 + 133
	9   UIKit                               0x00000001062997c0 -[_UIViewControllerTransitionContext completeTransition:] + 110
	10  UIKit                               0x0000000105bc51d3 -[UIViewAnimationBlockDelegate _didEndBlockAnimation:finished:context:] + 326
	11  UIKit                               0x0000000105bac136 -[UIViewAnimationState sendDelegateAnimationDidStop:finished:] + 209
	12  UIKit                               0x0000000105bac46c -[UIViewAnimationState animationDidStop:finished:] + 76
	13  QuartzCore                          0x00000001059ca882 _ZN2CA5Layer23run_animation_callbacksEPv + 308
	14  libdispatch.dylib                   0x00000001086f3614 _dispatch_client_callout + 8
	15  libdispatch.dylib                   0x00000001086dba1c _dispatch_main_queue_callback_4CF + 1664
	16  CoreFoundation                      0x0000000104fbf1f9 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 9
	17  CoreFoundation                      0x0000000104f80dcb __CFRunLoopRun + 2043
	18  CoreFoundation                      0x0000000104f80366 CFRunLoopRunSpecific + 470
	19  GraphicsServices                    0x0000000108bf4a3e GSEventRunModal + 161
	20  UIKit                               0x0000000105b4c900 UIApplicationMain + 1282
	21  Today                               0x00000001031c6307 main + 135
	22  libdyld.dylib                       0x0000000108728145 start + 1
	23  ???                                 0x0000000000000001 0x0 + 1
)
libc++abi.dylib: terminating with uncaught exception of type NSException
```

**Comment by arittr** [comment]
hey @jamztang - i believe this is due to our library not playing particularly nice when integrated as a sub dependency. the SDK assumes the Storyboard to be in the main app bundle (which here it is not) - you can copy the resource over manually or change a line of code to point to the correct location. see #207 and PRQ #250 for the relevant code changes - i personally think copying the resource over manually is the best route here.

**Comment by samgreen** [comment]
@jamztang It looks like the podspec should properly copy over the storyboard files. Are you still experiencing this issue? Can you provide a Podfile that reliably reproduces the issue you're having?

**Comment by jamztang** [comment]
I didn't manage to try on a fresh project yet but this is what I'm specifying in my Swift application

```
use_frameworks!
inhibit_all_warnings!

target "Today" do
  pod "ARAnalytics", '~> 3.6.1', :subspecs => ["Mixpanel", "Countly", "ParseAnalytics"]
  pod 'JazzHands', '~> 0.2.1'
  pod "Spring"
end
```

This is Podfile.lock
```
PODS:
  - ARAnalytics/CoreIOS (3.6.2)
  - ARAnalytics/Countly (3.6.2):
    - ARAnalytics/CoreIOS
    - Countly
  - ARAnalytics/Mixpanel (3.6.2):
    - ARAnalytics/CoreIOS
    - Mixpanel
  - ARAnalytics/ParseAnalytics (3.6.2):
    - ARAnalytics/CoreIOS
    - Parse
  - Bolts/Tasks (1.2.0)
  - Countly (3.0.0)
  - JazzHands (0.2.1)
  - Mixpanel (2.8.1):
    - Mixpanel/Mixpanel (= 2.8.1)
  - Mixpanel/Mixpanel (2.8.1):
    - Mixpanel/MPCategoryHelpers
  - Mixpanel/MPCategoryHelpers (2.8.1)
  - Parse (1.7.5):
    - Bolts/Tasks (>= 1.2.0)
  - Spring (1.0.3)

DEPENDENCIES:
  - ARAnalytics/Countly (~> 3.6.1)
  - ARAnalytics/Mixpanel (~> 3.6.1)
  - ARAnalytics/ParseAnalytics (~> 3.6.1)
  - JazzHands (~> 0.2.1)
  - Spring

SPEC CHECKSUMS:
  ARAnalytics: 697797d915d0afadf27f79563b262557cbe3d378
  Bolts: d176cb1e0012175589e389a9db49b85e27787576
  Countly: 4212967fc5739ffccfbff3b0fe863be45c984879
  JazzHands: d2343c3f2f9d70598d0cad8f521ce3893842a44a
  Mixpanel: 93c4825025e6380ced273ed7178496282e385077
  Parse: 7c0f115eff3e0cea3fac08d50bda47154edb8832
  Spring: 3f02b5dc3308572ef177b3cad0fa7bbc4b0706f7

COCOAPODS: 0.37.2
```

**Comment by samgreen** [comment]
This will be resolved in v2.8.2

**Comment by samgreen** [comment]
This should be resolved now if you want to do a `pod install`.
